### PR TITLE
chore: update author in package.json

### PR DESCRIPTION
**Pull request by benoitdeziel**
## Description

This PR updates the `author` field in the package.json of each project

## Checklist

- [ ] ~Updated the version number in the `package.json`~
- [ ] ~Test any new configuration on a repo using [`npm link`](https://docs.npmjs.com/cli/link)~

### CODE REVIEW

**Review by davidmartinezmarin — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -64,10 +64,6 @@ Next, add the following configuration to the `.vscode/settings.json` file for a
 
 > Note: If you'd like you can also add these settings to your VS Code user settings so the settings will apply to all your projects.
 
-## Pre-commit linting with Husky & lint-staged
-
-TODO
```
Comment (davidmartinezmarin) [inline README.md:69]: tu pourrais ajouter une petit tache pour documenter ?

Comment (benoitdeziel) [reply README.md:69]: yep

**Comment by benoitdeziel** [comment]
:rocket: PR was released in [`eslint-config-landr@0.5.0`](https://github.com/LandrAudio/linting-and-formatting/releases/tag/eslint-config-landr%400.5.0), [`prettier-config-landr@0.2.0`](https://github.com/LandrAudio/linting-and-formatting/releases/tag/prettier-config-landr%400.2.0), [`stylelint-config-landr@0.1.0`](https://github.com/LandrAudio/linting-and-formatting/releases/tag/stylelint-config-landr%400.1.0) :rocket:

**Pull request merged**